src/processing/mode/android/AndroidBuild.java

```diff
@@ -179,6 +181,8 @@ public File createProject() throws IOException, SketchException {
 //      InputStream input = PApplet.createInput(getCoreZipLocation());
 //      PApplet.saveStream(new File(libsFolder, "processing-core.jar"), input);
       Base.copyFile(coreZipFile, new File(libsFolder, "processing-core.jar"));
+      
+      copySupportV4(libsFolder);
```

Comment:
Fragments are not supported in Gingerbead. This library is added to make our sketch work on gingerbread devices. Gingerbread has 5.6% market share as of now ([See this](https://developer.android.com/about/dashboards/index.html?utm_source=suzunone)).
Also, this library is open source and is licensed under Apache Version 2.0 ([See this](https://developer.android.com/about/dashboards/index.html?utm_source=suzunone))

If we remove this jar, we'll only be able to support post gingerbread versions.